src/Chapter4.hs

```diff
@@ -259,7 +261,7 @@ name.
 
 > QUESTION: Can you understand why the following implementation of the
   Functor instance for Maybe doesn't compile?
-
+Needs to check for nothing 
```

Comment:
That's almost correct! In the implementation, the variable `x` has type `Maybe a` but the function needs to return `Maybe b`, so you can't just return `x`, the types don't match.